DemoQA/cypress/pages/bookstore/bookPage.js

```diff
@@ -0,0 +1,26 @@
+export const bookPage= {
+    TXT_SEARCH : "#searchBox",
+    TB_BOOK: '.ReactTable',
```

Comment:
You select the `a` tag not the table
![image](https://github.com/hoanghong72/WC3-Homework/assets/147176863/3d23098b-d4bd-4a6f-ace5-64cf5e4bb369)
You can use `a[href]`